setup.py

```diff
@@ -27,7 +27,9 @@ def get_version():
     'appdirs>=1.4.3,<2.0',
     'carto>=1.11.3,<2.0',
     'jinja2>=2.10.1,<3.0',
+    'numpy<=1.22.4',
     'pandas>=0.25.0',
+    'shapely>=1.7,<=1.7.1',
```

Comment:
Why not  `<2.0` ?. The current range is too restrictive

Reply:
> The first approach was to use <2.0 and that's enough to fix the reported issue. However, with that version, the following test was failing. To fix it, we would need to downgrade it to version 1.7.1. [Job](https://github.com/CartoDB/cartoframes/actions/runs/3759544118/jobs/6389235514)

The error is the following:
```
      def _get_coords(self):
          """Access to geometry's coordinates (CoordinateSequence)"""
  >       raise NotImplementedError("set_coords must be provided by derived classes")
  E       NotImplementedError: set_coords must be provided by derived classes
  
  .tox/py37/lib/python3.7/site-packages/shapely/geometry/base.py:338: NotImplementedError
```

It seems a breaking change in that method `_get_coords` that it is not implemented anymore in the base class; when version `>1.7.1`.

Reply:
This is affecting only the BaseGeometry in the test. Updating that, the range can be extended up to <2.0

Reply:
This is affecting only the BaseGeometry in the test. Updating that, the range can be extended up to <2.0